test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -196,7 +196,24 @@ string TestFunctionCall::format(
 			}
 		}
 
-		stream << formatGasExpectations(_linePrefix, _renderMode == RenderMode::ExpectedValuesActualGas, _interactivePrint);
+		stream << formatGasExpectations(_linePrefix, _renderMode == RenderMode::ExpectedValues && _gasRenderMode == GasRenderMode::ActualGas, _interactivePrint);
```

Comment:
The parameter is called `_useActualCost` and should really depend only on `_gasRenderMode`. The condition checked specifically for `ExpectedValuesActualGas` only because there was no `ActualValuesActualGas`.

```suggestion
		stream << formatGasExpectations(_linePrefix, _gasRenderMode == GasRenderMode::ActualGas, _interactivePrint);
```